prbt_support/test/acceptance_tests/acceptance_test_joint_limits.py

```diff
@@ -0,0 +1,237 @@
+#!/usr/bin/env python
+# Copyright (c) 2018 Pilz GmbH & Co. KG
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+
+import unittest
+from math import radians
+
+import rospy
+from urdf_parser_py.urdf import URDF
+from actionlib import SimpleActionClient
+from control_msgs.msg import FollowJointTrajectoryAction, FollowJointTrajectoryGoal, FollowJointTrajectoryResult
+from trajectory_msgs.msg import JointTrajectory, JointTrajectoryPoint
+from sensor_msgs.msg import JointState
+
+_CONTROLLER_ACTION_NAME = '/prbt/manipulator_joint_trajectory_controller/follow_joint_trajectory'
+_ACTION_SERVER_TIMEOUT_SEC = 5
+_JOINT_STATES_TOPIC_NAME = '/joint_states'
+_WAIT_FOR_MESSAGE_TIMEOUT_SEC = 1
+
+_JOINT_LIMITS_DEGREE = {
```

Comment:
add a comment that these are from the datasheet. otherwise they are just magic numbers.
maybe link to it: https://www.pilz.com/download/open/PRBT_6_Operat_Manual_1004685-EN-02.pdf (page 19)